src/common/snippets/src/lowered/pass/brgemm_blocking.cpp

```diff
@@ -146,21 +146,21 @@ bool BrgemmBlockingBase::mark_blocking_loops(snippets::lowered::LinearIR& linear
 
     const auto& loop_manager = linear_ir.get_loop_manager();
     if (!ov::snippets::utils::is_full_dim_value(k_block)) {
-        const std::vector<LoopPort> entries{LoopPort(brgemm_expr->get_input_port(0), true, 0),
-                                            LoopPort(brgemm_expr->get_input_port(1), true, 1)};
-        const std::vector<LoopPort> exits{LoopPort(brgemm_expr->get_output_port(0), false)};
+        const std::vector<LoopPort> entries{LoopPort::create<LoopPort::Type::Incremented>(brgemm_expr->get_input_port(0), 0),
+                                            LoopPort::create<LoopPort::Type::Incremented>(brgemm_expr->get_input_port(1), 1)};
+        const std::vector<LoopPort> exits{LoopPort::create<LoopPort::Type::NotProcessed>(brgemm_expr->get_output_port(0))};
```

Comment:
Minor: if we add `using Type = LoopPort::Type;` then we won't have to write `LoopPort::` every time.

In theory, we can employ `std::integral_constant` so every value from enum would be converted to a type. It's much more convenient to work with types (rather than non-type args) when you are using templates.
Anyway, I'm just thinking out loud, we shouldn't do it now. It's better to introduce separate loop port types.

Reply:
Added  `using Type = LoopPort::Type;` to affected files. Thanks for the idea!